app/src/request-templates.ts

```diff
@@ -66,7 +68,11 @@ export const useProofRequestTemplates = (useDevRestrictions: boolean) => {
           {
             schema: personSchema,
             requestedAttributes: [
-              { names: ['given_names', 'family_name', 'picture'], restrictions: verifiedPersonRestrictions },
+              {
+                names: ['given_names', 'family_name', 'picture'],
+                restrictions: verifiedPersonRestrictions,
+                nonRevoked: { to: now() },
```

Comment:
If a user has first opened the wallet and then afterwards a credential is revoked, will it be accepted because it's after the `now()`?

Reply:
now() gets called whenever the template is fetched, so any time the QR code is refreshed the time will be updated. I've tested the changes and it seems to be working as expected, if a credential is revoked before receiving a proof then it will not be valid. I've added the functionality so support this in my bifold PR.